igvc/launch/imu.launch

```diff
@@ -1,4 +1,4 @@
 <!-- imu.launch -->
 <launch>
-    <node name="imu" pkg="igvc" type="imu"/>
+    <node name="imu" pkg="igvc" type="imu" output="screen" />
```

Comment:
If `output="screen"` was added for debugging, it should be removed before merging

Reply:
`output="screen"` is common for roslaunch nodes even when not debugging. Without this attribute, ROS logging macros (ie ROS_INFO_STREAM) write to a log file instead of the terminal.
See http://wiki.ros.org/roslaunch/XML/node#Attributes